src/functions.c

```diff
@@ -118,10 +118,7 @@ dictType librariesDictType = {
 /* Dictionary of engines */
 static dict *engines = NULL;
 
-/* Libraries Ctx.
- * Contains the dictionary that map a library name to library object,
- * Contains the dictionary that map a function name to function object,
- * and the cache memory used by all the functions */
```

Comment:
we are missing the other field engines_stats, since we do have comment in functionsLibCtx, i think we can drop this comment.